plugins/release-manager/src/ReleaseManagerPage.test.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { render, act } from '@testing-library/react';
+import { wrapInTestApp } from '@backstage/test-utils';
+import ReleaseManagerPage from './ReleaseManagerPage';
+import useAxios from 'axios-hooks';
+import { tracksMock } from './utils/mocks';
+
+jest.mock('@backstage/core', () => {
```

Comment:
Use this pattern instead for 98% fewer headaches: https://github.com/backstage/backstage/blob/7647449390e9167c6dee247ce67d570bb7181761/packages/test-utils/src/testUtils/appWrappers.test.tsx#L108 😅 

In general there should be very few reasons to use jest module mocks. The backstage is is structured to allow for more direct injection patterns, and encourages those patterns within plugins as well.

Reply:
Yeah, the tests haven't been updated for a while. We have been migrating the plugin to keep up with the changes over a couple of months. Thanks for the example, will take care of it 👍 !